src/main/java/com/iota/iri/service/tipselection/impl/TipSelectorConflux.java

```diff
@@ -56,25 +56,50 @@ public TipSelectorConflux(Tangle tangle,
 
     @Override
     public List<Hash> getTransactionsToApprove(int depth, Optional<Hash> reference) throws Exception {
+
+        long start = System.currentTimeMillis();
+
         List<Hash> tips = new LinkedList<>(); 
 
         // Parental tip
         Hash parentTip = tangle.getLastPivot();
         tips.add(parentTip);
 
+        long end = System.currentTimeMillis();
```

Comment:
remove these times

Reply:
Be careful System.currentTimeMillis().The granularity of the value depends on the underlying operating system and may be larger. If someone modifies the system time, it will be affected.

Reply:
Thanks! Done removing these times for debugging.